actors/builtin/miner/miner_test.go

```diff
@@ -566,6 +566,49 @@ func TestExtendSectorExpiration(t *testing.T) {
 	})
 }
 
+func TestAddLockedFund(t *testing.T) {
```

Comment:
While we should have some test for the miner actor's `AddLockedFund`, I think you should be adding tests to `miner_state_test` instead, which already has a lot of tests for the vesting table. Please add to `TestVesting_AddLockedFunds_Table` instead, perhaps pulling the period boundary into the table entries.